pkg/agent/openflow/network_policy.go

```diff
@@ -793,7 +846,7 @@ func (c *clause) addAddrFlows(featureNetworkPolicy *featureNetworkPolicy, addrTy
 func (c *clause) addServiceFlows(featureNetworkPolicy *featureNetworkPolicy, ports []v1beta2.Service, priority *uint16, matchSrc bool) []*conjMatchFlowContextChange {
 	var conjMatchFlowContextChanges []*conjMatchFlowContextChange
 	for _, port := range ports {
-		matches := generateServicePortConjMatches(c.ruleTable.GetID(), port, priority, featureNetworkPolicy.ipProtocols, matchSrc)
+		matches := generateServiceConjMatches(c.ruleTable.GetID(), port, priority, featureNetworkPolicy.ipProtocols, matchSrc)
```

Comment:
Maybe change port it to service? Also I don't understand the comments for this func, what is `NetworkPolicyPorts`?

Reply:
Changed. 
`NetworkPolicyPort` represents `ports` field in Antrea-native NP. Before this ICMP PR, in controlplane, `NetworkPolicyPort` will be translated to Antrea `Service` in a 1-1 relation. I changed `NetworkPolicyPorts` to Antrea `Service` in the comment. Hope it could avoid confusion.

Reply:
Thanks for explaining this.